corehq/apps/app_manager/app_translations/generators.py

```diff
@@ -15,12 +15,18 @@
 
 
 class POFileGenerator:
-    def __init__(self, domain, app_id, version, key_lang, source_lang, lang_prefix):
+    def __init__(self, domain, app_id, version, key_lang, source_lang, lang_prefix,
+                 exclude_if_default=False):
```

Comment:
Could you add a docstring to this class at some point?  Doesn't need to be in this PR, just thinking that without context, it might not be clear what these params mean.

Reply:
Added.